public/style.css

```
@@ -447,7 +455,7 @@ div, form, h3, h4, select, button {
   text-decoration: none;
 }
 
-.no-time-slot-wrapper {
+.no-time-slot-wrapper, .sucessful-booking-wrapper {
```

Spelling error... quick fix
```suggestion
.no-time-slot-wrapper, .successful-booking-wrapper {
```

---

make sure you also update the spelling everywhere this class is used